typescript-react/src/components/View/Grid.tsx

```diff
@@ -211,7 +211,7 @@ export class GridComponent<Row> extends React.PureComponent<IGridComponent<Row>,
       return CellType.Boolean;
     }
     if (typeof value === 'number') {
```

Comment:
what does this mean? it doesn't look right

Reply:
There is a comment above this function:
```
A very primitive inference mechanism for cells that come from a Map. In general, this should not be used (use a typed grid or a ResultSet instead), but it can stay for now.
```
I didn't want to break compatibility so I just returned `CellType.Integer` here since it will behave the same as `CellType.Number` did before. Should I do something else?

Reply:
But what if its decimal number? will it show decimals with that format?

Reply:
Good point, this won't show decimals. Will fix.